actionpack/lib/action_dispatch/http/upload.rb

```diff
@@ -59,6 +59,11 @@ def path
         @tempfile.path
       end
 
+      # Shortcut for +tempfile.to_path+.
+      def to_path
+        @tempfile.to_path
+      end
```

Comment:
You should add a simple delegation test for `#to_path` to `actionpack/test/dispatch/uploaded_file_test.rb` in order for this to be mergeable.

Reply:
👍Thanks! I'll get to it a little later today 😄